openstack/networking/v1/eips/results.go

```diff
@@ -30,6 +30,7 @@ type PublicIp struct {
 	BandwidthSize      int    `json:"bandwidth_size"`
 	BandwidthShareType string `json:"bandwidth_share_type"`
 	IpVersion          int    `json:"ip_version"`
+	Name               string `json:"alias"`
```

Comment:
Maybe change it to `alias`? Now we have 2 `name` attributes in `BandwidthOpts` and `PublicIpOpts` which are difficult to distinguish.